AnnoDesigner/Helper/CoordinateHelper.cs

```diff
@@ -157,5 +158,15 @@ public Rect Rotate(Rect rect)
 
             return new Rect(new Point(xPrime, yPrime), Rotate(rect.Size));
         }
+
+        /// <summary>
+        /// Rotates the given GridDirection object 90 degrees clockwise.
+        /// </summary>
+        /// <param name="direction"></param>
+        /// <returns></returns>
+        public GridDirection Rotate(GridDirection direction)
```

Comment:
Does this still work if `Undefined` would be added?
Maybe implement it as a `switch`.